tests/foreman/api/test_templatesync.py

```diff
@@ -423,11 +423,16 @@ def test_positive_export_filtered_templates_to_localdir(
             template['exported'] for template in exported_temps['message']['templates']
         ].count(True)
         assert exported_count == 23
-        assert ssh.command(f'find {dir_path} -type f -name *ansible* | wc -l').stdout[0] == '23'
+        assert (
+            default_sat.execute(
+                f'find {dir_path} -type f -name *ansible* | wc -l'
+            ).stdout.splitlines()[0]
```

Comment:
I think you can drop the `splitlines()[0]` here, as the `|wc -l` will reduce output to one line.

Reply:
Applies to all the uses in this file. Understand if you want to keep it really explicit though.

Reply:
I think I can drop the splitlines if I include `strip`